server/routerlicious/packages/tinylicious/src/services/tenantManager.ts

```diff
@@ -22,7 +28,19 @@ export class TinyliciousTenant implements ITenant {
 		private readonly url: string,
 		private readonly historianUrl: string,
 	) {
-		const historian = new Historian(historianUrl, false, false);
+		// Using an explicitly constructed rest wrapper so we can pass the Axios instance whose static defaults
+		// were modified by t9s, and avoid issues if the module that contains BasicRestWrapper depends on a different
```

Comment:
```suggestion
		// were modified by Tinylicious, and avoid issues if the module that contains BasicRestWrapper depends on a different
```

I really think we should avoid using abbreviations of this form. IMO they just reduce accessibility.